man/man8/zfs.8

```diff
@@ -268,6 +268,21 @@ zfs \- configures ZFS file systems
 .LP
 .nf
 \fBzfs\fR \fBdiff\fR [\fB-FHt\fR] \fIsnapshot\fR \fIsnapshot|filesystem\fR
+.fi
+
+.LP
+.nf
+\fB\fBzfs key -l\fR \fIfilesystem\fR | \fIvolume\fR\fR
```

Comment:
I had the same thoughts... nothing better jumps to mind but I'll keep pondering...

FWIW, I think that the tedious `zfs loadkey` etc are in keeping with the existing interface, which does have some collections of verbs that all operate on the same concepts (e.g. `zfs allow`, `zfs unallow`; and `zfs hold`, `zfs release`, `zfs holds`).